src/test_handler.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from src.handler import _get_log_type
+
+def test_get_log_type_with_parameter():
+    assert _get_log_type("PARAMETER") == "PARAMETER"
+
```

Comment:
Should we introduce a test where we don't pass any parameter and we don't set any environment variable, to check that it returns `""` correctly?

Reply:
Yes, I think it's a good idea. :)